config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
Thanks! I think you should probably bump `formatVersion` (https://github.com/clearcontainers/runtime/blob/master/cc-env.go#L33) as adding the machine type is an output format change.